plugins/kubernetes-backend/src/service/router.ts

```diff
@@ -98,10 +100,17 @@ export async function createRouter(
     'kubernetes.clusterLocatorMethods',
   ) as ClusterLocatorMethod[];
 
-  const clusterDetails = await getCombinedClusterDetails(
-    clusterLocatorMethods,
-    options.config,
-  );
+  let clusterDetails: ClusterDetails[];
+
+  if (options.clusterSupplier) {
+    clusterDetails = await options.clusterSupplier.getClusters();
+    // Append supplied clusters to cluster retrieved via getCombinedClusterDetails
```

Comment:
What's the meaning of this comment? Nothing is appended

Reply:
oops - that comment was correct before I cleaned up our internal version a bit - let me take it out.